### DISCUSSION TITLE
when there is only one message, how to make it on the top, not bottom

### DISCUSSION THREAD

**Discussion by dodoto** [Shopify/flash-list#665] [Q&A]
When there is only one message, it render on bottom of screen. But I want it display on top.
In FlatList, it works on below code
```
<View style={{flex: 1}}>
  <View>
    <FlatList inverted/>
  </View>
</View>
```
But in FlashList, I get a warn
```
FlashList's rendered size is not usable. Either the height or width is too small (<2px). Please make sure that the parent view of the list has a valid size. FlashList will match the size of the parent.
```
Is there any way to make message render from top, when content size is smaller than container size?

**Comment by naqvitalha** [comment]
Can you try inverting the list in that specific case to see if it works for you? You need to set `inverted={true}` on the list.

**Reply by dodoto** [reply]
It works by toggle inverted. But unfortunally, there is a problem. Inverted is false at first, when content size is larger than layout size, toggle inverted. And you will see this
![inverted](https://user-images.githubusercontent.com/45391750/200452713-3f0bebba-0244-4733-80ee-95c27202c8f6.png).
After next input and send, it will auto fix.

**Reply by hongppa324** [reply]
@dodoto Hi, did you solve this problem??
